go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/lithammer/dedent v1.1.0
 	github.com/stretchr/testify v1.8.2
-	k8s.io/api v0.26.3
-	k8s.io/apiextensions-apiserver v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
-	k8s.io/code-generator v0.26.3
+	k8s.io/api v0.27.1
+	k8s.io/apiextensions-apiserver v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v0.27.1
+	k8s.io/code-generator v0.27.1
 	k8s.io/klog/v2 v2.90.1
-	k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/utils v0.0.0-20230209194617-a36077c30491
+	sigs.k8s.io/controller-runtime v0.13.1-0.20230417175046-71885c9b8a6b
```

Comment:
just pointing out since we're in the same boat in contour, this is an unreleased version

Reply:
this will be upgraded by bot when controller-runtime release new version.

Reply:
That sounds alright, I've created a follow-up issue as well so that we can't forget (and also includes some other maintenance items to check before GA): https://github.com/kubernetes-sigs/gateway-api/issues/1970